pkgs/development/python-modules/matrix-nio/default.nix

```diff
@@ -50,7 +65,20 @@ buildPythonPackage rec {
     atomicwrites
   ];
 
-  doCheck = false;
+  checkInputs = [
+    faker
+    aioresponses
+    hypothesis
+    pytest-aiohttp
+    pytest-benchmark
+  ];
+
+  disabledTests = [
+    # touches network
+    "test_connect_wrapper"
+    # time dependent and flaky
+    "test_transfer_monitor_callbacks"
+  ];
 
```

Comment:
```suggestion

  pythonImportsCheck = [ "nio" ];

```

Reply:
Oh one can have both normal checks and pythonImportsCheck? TIL